app/views/accounts/_pii.html.erb

```diff
@@ -5,9 +5,7 @@
         <div class="usa-alert__text">
           <%= t('account.re_verify.banner') %>
           <%= render ButtonComponent.new(
-                action: ->(**tag_options, &block) do
-                  button_to(account_reauthentication_path, **tag_options, &block)
-                end,
```

Comment:
Looks like on this branch, after these changes, there are 26 `action:` usages left, would we consider removing all of them in a follow-up PR?
```
identity-idp:aduth-backup-code-s-form-link> git grep -A 3 ButtonComponent.new -- app | grep action: | wc -l
      26
```

Reply:
Yeah I can! I actually manually un-edited some additional files since they weren't strictly required to update, so as to keep review simpler. But definitely want to get everything migrated over.

Reply:
Follow-up at #10468